app/repo/pg/user.go

```diff
@@ -0,0 +1,29 @@
+// Package repo represents adapter layer
```

Comment:
The same about naming - lets keep "repository", "transport" etc.

Reply:
ok. no problem